benchmark/http/_http_simple.js

```diff
@@ -4,10 +4,10 @@ var http = require('http');
 
 var port = parseInt(process.env.PORT || 8000);
 
-var fixed = 'C'.repeat(20 * 1024),
-  storedBytes = {},
-  storedBuffer = {},
-  storedUnicode = {};
+var fixed = 'C'.repeat(20 * 1024);
+var storedBytes = Object.create(null);
+var storedBuffer = Object.create(null);
+var storedUnicode = Object.create(null);
```

Comment:
Right, but I seem to recall some issue with that benchmark. I think there was discussion in some issue/PR awhile ago about it?